### PR TITLE
save only ipaddress to the uploaded file server

### PR DESCRIPTION
**Pull request by Ariful-Islam-19**

### CODE REVIEW

File: dicoveryserver/save_uploaded_info.php [hunk 1]
```diff
@@ -10,22 +10,36 @@
...
+// }
+function save_resource($conn, $userid, $ipAddress)
 {
     $query = "INSERT INTO `resource` (`resource_name`, `user_id`, `file_type`, `date`, `file_name`, `ip_address`)
               VALUES (?, ?, ?, CURDATE(), ?, ?)";
 
     $stmt = $conn->prepare($query);
 
-    $stmt->bind_param("sisss", $resourceName, $userid, $fileType, $fileName, $ipAddress);
+    $stmt->bind_param("is",  $userid, $ipAddress);
```
Comment (maifeeulasad) [inline dicoveryserver/save_uploaded_info.php:34]: what is `sisss` and `is`?

File: dicoveryserver/save_uploaded_info.php [hunk 2]
```diff
@@ -10,22 +10,36 @@
 $ipAddress = $_POST['ipAddress'] ?? null;
 
 // Function to save resource
-function save_resource($conn, $userid, $fileName, $fileType, $resourceName, $ipAddress)
+// function save_resource($conn, $userid, $fileName, $fileType, $resourceName, $ipAddress)
```
Comment (maifeeulasad) [inline dicoveryserver/save_uploaded_info.php:13]: commented out code, looks really nice

File: all_share.php
Comment (maifeeulasad) [inline all_share.php:1]: Commented out code!! But why? If you don't need it just remove it

**Review by maifeeulasad — CHANGES_REQUESTED** [review]
Please complete them ASAP. They shouldn't take more than 10 minutes.

File: dicoveryserver/save_uploaded_info.php (hunk 1 above)
Comment (Ariful-Islam-19) [reply dicoveryserver/save_uploaded_info.php:34]: The strings "sisss" and "is" in the $stmt->bind_param() function in PHP represent the data types of the parameters being bound to the prepared SQL statement,

"sisss" indicates the data types for five parameters in the order they appear:
s: The first parameter is a string ($resourceName).
i: The second parameter is an integer ($userid).
s: The third parameter is a string ($fileType).
s: The fourth parameter is a string ($fileName).
s: The fifth parameter is a string ($ipAddress).

**Pull request merged**